path_2023.py

```diff
@@ -31,6 +35,16 @@ def run() -> int:
     if not result:
         print("ERROR: load_waypoint_name_to_coordinates_map")
         return -1
+    
+    result, waypoints_list = waypoints_dict_to_list.waypoints_dict_to_list(waypoint_name_to_coordinates)
+    if not result:
+        print("ERROR: Unable to convert waypoints from dict to list")
+        return -1
+    
+    result  = waypoints_to_kml.waypoints_to_kml(waypoints_list, KML_FILE_NAME, KML_FILE_PATH)
```

Comment:
The common repository was updated, which changed the return portion of the function signature. Can you submodule update and then update this to align? Sorry about that.